src/ifcopenshell-python/ifcopenshell/geom/occ_utils.py

```diff
@@ -30,85 +30,93 @@
 handle, main_loop, add_menu, add_function_to_menu = None, None, None, None
 
 DEFAULT_STYLES = {
```

Comment:
this change might also be undesired

Reply:
Well, it's not a big deal to me. These aligned indents are a pain to maintain anyway when updates take place.